### ISSUE TITLE
Clean up Old Paragraph Data

### ISSUE THREAD

**Issue by indytechcook** [department-of-veterans-affairs/va.gov-cms#3593]
## Description
After the fixing issues with data clean up, there are leftover paragraphs and paragraph revisions.

**Example paragraph_revision not associated to the same node as other revisions.**

The node 779 has a single field which has the paragraph 
```
select * from node__field_content_block where entity_id=779;

| field_content_block_target_id | field_content_block_target_revision_id |
|-------------------------------|----------------------------------------|
| 5331                          | 35411                                  |

```

The following query shows all of the revisions which are associated with the paragraph.  There are several which are not associated with the correct `parent_id` of 779.

`select * from paragraphs_item_revision_field_data where id=5331 AND parent_id <> 779;`

All of the revisions should be removed.

Related Issues:
* #3449 
* #3450 
* #3591

## Acceptance Criteria
- [ ] All paragraphs which are not associated to a field.
- [ ] All paragraph_revisions which do not point to the current parent entity should be deleted.

**Comment by ndouglas** [comment]
These are the source nodes that were cloned, causing #3450.  Like #3591, this can be repaired with a modification of the script that closed #3450.  Just to be safe, I'm going to tackle this after #3591.

**Comment by cmaeng** [comment]
this work was completed as part of other issues. closing to no action.
